virtual_ecosystem/data_variables.toml

```
@@ -1,550 +1,727 @@
+description = "Mean mixing length"
+name = "mean_mixing_length"
+unit = "m"
+variable_type = "float"
 
 [[variable]]
+axis = ["space"]
 description = "Temperature-dependent molar density of air"
-initialised_by = "abiotic"
 name = "molar_density_air"
 unit = "kg m-3"
-updated_by = "abiotic"
-used_by = ["abiotic"]
-
-[[variable]]
-description = "Net shortwave radiation at the surface"
-initialised_by = "abiotic"
-name = "netradiation_surface"
-unit = "J m-2"
-updated_by = "abiotic"
-used_by = ["abiotic"]
+variable_type = "float"
 
 [[variable]]
+axis = []
 description = "Plant net CO2 assimilation"
-initialised_by = "plants"
 name = "plant_net_co2_assimilation"
 unit = "ppm"
-updated_by = "plants"
-used_by = ["abiotic", "plants"]
+variable_type = ""
 
 [[variable]]
-description = "Top of canopy photosynthetic photon flux density"
-initialised_by = "abiotic"
-name = "ppfd"
-unit = "mol m-2"
-updated_by = "abiotic"
-used_by = ["abiotic", "plants"]
+axis = ["space", "time"]
+description = "Top of canopy photosynthetic photon flux density (PPFD)"
+name = "photosynthetic_photon_flux_density"
+unit = "µmol m-2 s-1"
+variable_type = "float"
 
 [[variable]]
-description = "Precipitation"
-initialised_by = "external"
+axis = ["space", "time"]
+description = "Precipitation input at the top of the canopy"
 name = "precipitation"
 unit = "mm"
-updated_by = ""
-used_by = ["hydrology"]
+variable_type = "float"
 
 [[variable]]
-used_by = [ "hydrology", "plants]
+axis = ["space"]
+description = "Precipitation that reaches surface"
+name = "precipitation_surface"
+unit = "mm"
+variable_type = "float"
 
 [[variable]]
-name = "relative_humidity"
+axis = ["space"]
 description = "Relative humidity profile"
+name = "relative_humidity"
 unit = "%"
-initialised_by = ["abiotic", "abiotic_simple"]
-updated_by = ["abiotic", "abiotic_simple"]
-used_by = ["abiotic", "abiotic_simple", "hydrology", "plants"]
+variable_type = "float"
 
 [[variable]]
-name = "relative_humidity_ref"
+axis = ["space", "time"]
 description = "Relative humidity at reference height (2m)"
+name = "relative_humidity_ref"
 unit = "%"
-initialised_by = "external"
-updated_by = ""
-used_by = ["abiotic", "abiotic_simple", "hydrology"]
+variable_type = "float"
+
+[[variable]]
+axis = ["space"]
+description = "Relative turbulence intensity"
+name = "relative_turbulence_intensity"
+unit = "-"
+variable_type = "float"
 
 [[variable]]
-name = "river_discharge"
-description = "River discharge"
+axis = ["space"]
+description = "River discharge rate"
+name = "river_discharge_rate"
 unit = "m3 s-1"
-initialised_by = "hydrology"
-updated_by = "hydrology"
-used_by = [ "hydrology"]
+variable_type = "float"
 
 [[variable]]
-name = "stream_flow"
-description = "Estimated stream flow"
-unit = "mm per time step"
-initialised_by = "hydrology"
-updated_by = "hydrology"
-used_by = [ "hydrology"]
+axis = ["space"]
+description = "Aerodynamic resistance surface"
+name = "aerodynamic_resistance_surface"
+unit = "m2"
```

```suggestion
unit = "kg m2 s–3"
```
The unit is Ohm or kg m2 s–3